### PR TITLE
[Wear] Adds previews to queue and podcasts

### PR DESCRIPTION
**Pull request by kul3r4**
- Adds previews to queue and podcasts
- Fixes README which had some redundant `[` `]`

### CODE REVIEW

File: Jetcaster/wear/src/main/java/com/example/jetcaster/PreviewData.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ */
+
+package com.example.jetcaster
+
+import com.example.jetcaster.core.model.PlayerEpisode
+import com.example.jetcaster.core.model.PodcastInfo
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+val PreviewPodcasts = listOf(
```
Comment (yongsuk44) [inline Jetcaster/wear/src/main/java/com/example/jetcaster/PreviewData.kt:24]: Using [PreviewParameterProvider](https://developer.android.com/reference/kotlin/androidx/compose/ui/tooling/preview/PreviewParameterProvider) to explicitly supply parameters to Preview looks good.

Comment (kul3r4) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/PreviewData.kt:24]: thank you! Will take a look

Comment (kul3r4) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/PreviewData.kt:24]: done,thanks

**Review by yschimke — APPROVED** [review]

File: Jetcaster/wear/src/main/java/com/example/jetcaster/podcasts/PodcastsScreen.kt [hunk 2]
```diff
@@ -135,40 +93,114 @@ fun PodcastsScreen(
...
+            PodcastsScreenState.Empty ->
+                PodcastScreenEmpty(onDismiss)
+            PodcastsScreenState.Loading ->
+                PodcastScreenLoading(columnState)
+        }
+    }
+}
+
+@Composable
+fun PodcastScreenLoaded(
```
Comment (arriolac) [inline Jetcaster/wear/src/main/java/com/example/jetcaster/podcasts/PodcastsScreen.kt:110]: Nit: Make sure to add modifiers to these composables. Same in `QueueScreen`.

**Review by arriolac — APPROVED** [review]

File: Jetcaster/wear/src/main/java/com/example/jetcaster/podcasts/PodcastsScreen.kt (hunk 2 above)
Comment (yschimke) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/podcasts/PodcastsScreen.kt:110]: Is it worth adding lint to fail on these?

Comment (yschimke) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/podcasts/PodcastsScreen.kt:110]: Actually maybe it won't for this, just based on order?

Comment (arriolac) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/podcasts/PodcastsScreen.kt:110]: > Is it worth adding lint to fail on these?

Error might be too aggressive—it should warn at the very least.

Also, if you are using live templates, typing "comp" in the editor will automatically create the following template for you so it's easy to add a composable with a default modifier (I know this was in AS preview and I think should be available now in the stable channel with jellyfish):

```
@Composable
fun <cursor here>(modifier: Modifier = Modifier) {
}
```

Comment (kul3r4) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/podcasts/PodcastsScreen.kt]: thanks, added

**Pull request merged**